src/workerd/jsg/modules.h

```diff
@@ -7,10 +7,12 @@
 #include <kj/filesystem.h>
 #include <kj/map.h>
 #include <workerd/jsg/modules.capnp.h>
+#include <workerd/util/thread-scopes.h>
 #include <workerd/jsg/observer.h>
 #include <set>
 #include "function.h"
 #include "promise.h"
+#include <workerd/util/autogate.h>
```

Comment:
Super minor nit: can you keep the `<workerd/util/...>` includes together ;-)

Reply:
I did a small import cleanup here: I switched `function.h` and `promise.h` to fully qualified imports, sorted them all, and dropped `<set>` which seems to be unused.